uast/transformer/semantic.go

```diff
@@ -101,6 +101,19 @@ func CommentText(tokens [2]string, vr string) Op {
 		prefVar:    vr + "_pref",
 		suffVar:    vr + "_suff",
 		indentVar:  vr + "_tab",
+		doTrim:     false,
+	}
+}
+
+func CommentTextTrimmed(tokens [2]string, vr string) Op {
+	return &commentUAST{
```

Comment:
You can probably  do something like
```golang
op := CommentTextTrimmed(tokens, vr).(*commentUAST)
op.doTrim = true
```
or add an unexported `newCommentText` with the third argument.